com.unity.render-pipelines.universal/Documentation~/camera-component-reference.md

```diff
@@ -52,10 +52,15 @@ The Camera Inspector has the following sections when the Camera has its **Render
 <a name="base-rendering"></a>
 ## Rendering
 
+Note that when a Camera's *Render Mode* is set to *Base* and its *Render Target* is set to *Texture*, Unity does not expose the following properties in the Inspector for the Camera:
+
+* Allow Dynamic Resolution
+
 | __Property__               | __Description__                                              |
 | -------------------------- | ------------------------------------------------------------ |
 |__Renderer__ |Controls which renderer this Camera uses. |
 |__Post Processing__ |Enables post-processing effects. |
+|__Allow Dynamic Resolution__|Enables Dynamic Resolution rendering for this Camera.<br/>This property only appears when you select **Screen** from the **Output Target** drop-down.|
```

Comment:
Note: there are some UX implications when moving this setting to Rendering. As stated above the dynamic resolution setting is hidden once Render Target is `Texture`. Today these setting lie next to each other, if you move you might cause confusion. 
 
 URP also groups all settings that affect camera target properties in Output category. I'm not convinced moving to Rendering is a better design.
 
 @TheoWong-pixel thoughts?

Reply:
The interesting part is in HDRP this setting being in Output is far from Rendering, which will also cause the same problem since it affects more rendering properties, such as TAA (in HDRP). This whole PR's purpose was to remove that confusion.
If this is gonna cause confusion in URP then we should completely rethink this.

What I recommend is to rethink this: Why is Core enforcing a grouping at all? if this is an issue, then we should not have Core pipeline enforce a group. Thus maybe the core team should just change the core setting instead, and keep the groupings in the pipelines since they behave differently?